### PR TITLE
mimxrt/machine_pin: Updated DRIVE_# constants.

### PR DESCRIPTION
**Pull request by alphaFred**
Updated DRIVE_# constants representing pin drive strength.

Signed-off-by: Philipp Ebensberger

### CODE REVIEW

**Comment by alphaFred** [comment]
@dpgeorge based on our discussion from #8313. On a side note, it would be really great if you would find the time to also take a look at @robert-hh PR #7494 allowing us to bring some documentation for the mimxrt port to mainline.

File: ports/mimxrt/machine_pin.c [hunk 1]
```diff
@@ -381,14 +381,14 @@ STATIC const mp_rom_map_elem_t machine_pin_locals_dict_table[] = {
...
 
-    { MP_ROM_QSTR(MP_QSTR_DRIVER_OFF), MP_ROM_INT(PIN_DRIVE_OFF) },
-    { MP_ROM_QSTR(MP_QSTR_POWER_0),    MP_ROM_INT(PIN_DRIVE_POWER_0) }, // R0 (150 Ohm @3.3V / 260 Ohm @ 1.8V)
-    { MP_ROM_QSTR(MP_QSTR_POWER_1),    MP_ROM_INT(PIN_DRIVE_POWER_1) }, // R0/2
-    { MP_ROM_QSTR(MP_QSTR_POWER_2),    MP_ROM_INT(PIN_DRIVE_POWER_2) }, // R0/3
-    { MP_ROM_QSTR(MP_QSTR_POWER_3),    MP_ROM_INT(PIN_DRIVE_POWER_3) }, // R0/4
-    { MP_ROM_QSTR(MP_QSTR_POWER_4),    MP_ROM_INT(PIN_DRIVE_POWER_4) }, // R0/5
-    { MP_ROM_QSTR(MP_QSTR_POWER_5),    MP_ROM_INT(PIN_DRIVE_POWER_5) }, // R0/6
-    { MP_ROM_QSTR(MP_QSTR_POWER_6),    MP_ROM_INT(PIN_DRIVE_POWER_6) }, // R0/7
+    { MP_ROM_QSTR(MP_QSTR_DRIVE_OFF),  MP_ROM_INT(PIN_DRIVE_OFF) },
```
Comment (dpgeorge) [inline ports/mimxrt/machine_pin.c:384]: Is this level really off, ie it does not drive the GPIO at all?  Or is it just a very weak drive?

Comment (alphaFred) [reply ports/mimxrt/machine_pin.c:384]: According to the documentation it turns the output driver off.

Comment (robert-hh) [reply ports/mimxrt/machine_pin.c:384]: Tested: It switches the driver off. With a 0.9 MOhm load and the value set to 1, the output  voltage drops to 0. when drive=Pin.DRIVE_OFF is set. With drive=Pin.DRIVE_0, the output voltage reads 3.28 V.

**Comment by dpgeorge** [comment]
Rebased and merged in 62cb2069bb107f32efedcd59481de6373c854db1

File: ports/mimxrt/machine_pin.c (hunk 1 above)
Comment (dpgeorge) [reply ports/mimxrt/machine_pin.c:384]: Great, thanks for testing!

**Pull request closed without merging**